### PR TITLE
chore: updated my profile

### PR DESCRIPTION
**Pull request by eddiejaoude**
Updated as part of a YouTube video

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1779"></a>

### CODE REVIEW

File: public/data/eddiejaoude.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Eddie Jaoude",
   "type": "personal",
-  "bio": "Open Source DevRel | Founder of EddieHub",
+  "bio": "Open Source DevRel | Founder of EddieHub| GitHub Star",
```
Comment (Panquesito7) [inline public/data/eddiejaoude.json:4]: Hmm, what do you think if we add an extra space here? 🙂
```suggestion
  "bio": "Open Source DevRel | Founder of EddieHub | GitHub Star",
```

Comment (eddiejaoude) [reply public/data/eddiejaoude.json:4]: thank you 👍

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]
Nice! Thanks. 🎉

**Pull request merged**